src/Paprika.Benchmarks/SlottedArrayBenchmarks.cs

```diff
@@ -191,6 +246,59 @@ public int EnumerateNibble(byte nibble)
         return length;
     }
 
+    [Benchmark]
+    public bool Defragmentation()
+    {
+        var map = CreateMapForDefragmentation();
+
+        // Insert additional key to trigger defragmentation
+        var additionalKey = Keccak.Zero;
+        Span<byte> value = stackalloc byte[32];
+
+        for (byte i = 0; i < 32; i++)
+        {
+            value[i] = i;
+        }
+
+        return map.TrySet(NibblePath.FromKey(additionalKey), value);
```

Comment:
This performs just one set while the `CreateMap...` copies the page. Also, no keys are deleted. Is it a baseline?

Reply:
One key is already deleted as part of the setup of this benchmark. This one directly triggers the defragment operation as part of TrySet. I believe this should serve as a good baseline for the `Large` benchmark. 

 `Large` benchmark also sets only one key, but it deletes multiple random keys based on the parameter. So in effect  `Large` one is capturing [key(s) deletion + defrag + one Set] time, while the smaller one is capturing [defrag + one Set] time.